{"name": "test_01[1]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 3 == 1", "trace": "self = <test_case_04.Test_abc_03 object at 0x109b19a90>, before = 1\n\n    def test_01(self,before):\n        print(\"----->>>>我是测试方法<<<<----\");\n>       assert 3 == before;\nE       assert 3 == 1\n\ntest_case_04.py:24: AssertionError"}, "parameters": [{"name": "before", "value": "1"}], "start": 1610631193482, "stop": 1610631193482, "uuid": "6c191a4d-5eb0-4e5f-8556-7c1ee8ac331b", "historyId": "a97b05a160f1b794b20f95f71298aa3c", "testCaseId": "f7eb19b8d4e9053f77f9cc806fb477e7", "fullName": "test_case_04.Test_abc_03#test_01", "labels": [{"name": "suite", "value": "test_case_04"}, {"name": "subSuite", "value": "Test_abc_03"}, {"name": "host", "value": "yantianpeng.local"}, {"name": "thread", "value": "28222-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case_04"}]}